src/atoms/Icon/Icon.native.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import icons from '../../icons';
+
+const iconSize = {
+  s: 10,
+  m: 16,
+  l: 20,
+};
+
+const Icon = ({ size, name, fill }) => {
+  const IconComponent = icons[name];
+  return <IconComponent fill={fill} height={iconSize[size]} width={iconSize[size]} />;
+};
+
+Icon.defaultProps = {
+  size: 'l',
```

Comment:
Should we settle for a common default prop across components? For e.g, the default size is large here in Icon but in Link it is medium.

Reply:
Sure, makes sense.